test/test.cpp

```diff
@@ -1,15 +1,19 @@
+#include <cassert>
 #include <iostream>
 #include <string>
 
 #include "../dist/postject-api.h"
 
 int main() {
-  size_t size;
-  const void* ptr = postject_find_resource("foobar", &size, nullptr);
+  size_t size = 0;
 
-  if (ptr && size > 0) {
+  if (postject_has_resource() == true) {
+    const void* ptr = postject_find_resource("foobar", &size, nullptr);
+    assert(ptr && size > 0);
```

Comment:
I think this should be done with an if statement rather than an `assert`. We probably should be testing the final binary in both debug builds and release builds, and the `assert` will be removed when `-DNDEBUG` is set in a release build.